message_text_config.py

```diff
@@ -0,0 +1,99 @@
+# Prepare phase texts
+GLOBAL_LOAD_GAME = "Загрузка игры"
+GLOBAL_FIRST_TURN = "Первым будет ходить [активного плеера]"
```

Comment:
Please, change all strings to f-strings. (Add "f" letter before string definition)
Also, replace parts like [активного плеера] to smth like {active_player} - so it could be used in code.

Reply:
> Please, change all strings to f-strings. (Add "f" letter before string definition)
> Also, replace parts like [активного плеера] to smth like {active_player} - so it could be used in code.

We can use format instead of f-strings. I guess it will make easier to add some text like 'active_player'.

Reply:
Done